Packs/Okta/Integrations/Okta_v2/Okta_v2.py

```diff
@@ -783,8 +792,13 @@ def set_password_command(client, args):
     user_id = client.get_user_id(args.get('username'))
     password = args.get('password')
 
-    raw_response = client.set_password(user_id, password)
-    readable_output = f"{args.get('username')} password was last changed on {raw_response.get('passwordChanged')}"
+    if argToBoolean(args.get('one_time_password', False)):
+        raw_response = client.set_temp_password(user_id)
+        readable_output = f"The temporary password for {args.get('username')} is: {raw_response.get('tempPassword')}"
```

Comment:
@MosheEichler @idovandijk Do we want to print the password even if temporary to the playground ?

Reply:
I would not print it, but if it's already printing it in the current Okta version then I would not change that behavior.